embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/fakes/injection/FakeInitModule.kt

```diff
@@ -1,14 +1,17 @@
 package io.embrace.android.embracesdk.fakes.injection
 
 import io.embrace.android.embracesdk.fakes.FakeOpenTelemetryClock
+import io.embrace.android.embracesdk.fakes.FakeTelemetryService
 import io.embrace.android.embracesdk.injection.InitModule
 import io.embrace.android.embracesdk.internal.clock.Clock
 import io.embrace.android.embracesdk.internal.clock.NormalizedIntervalClock
 import io.embrace.android.embracesdk.internal.clock.SystemClock
 import io.embrace.android.embracesdk.internal.spans.EmbraceSpansService
 import io.embrace.android.embracesdk.internal.spans.SpansService
+import io.embrace.android.embracesdk.telemetry.TelemetryService
 
 internal class FakeInitModule(
     override val clock: Clock = NormalizedIntervalClock(systemClock = SystemClock()),
-    override val spansService: SpansService = EmbraceSpansService(clock = FakeOpenTelemetryClock(embraceClock = clock))
+    override val telemetryService: TelemetryService = FakeTelemetryService(),
```

Comment:
Can we use a regular `EmbraceTelemetryService` here? We use the `FakeInitModule` in the `IntegrationTestRule` and it would be good to able to inspect the logged telemetry in those tests.